libretro-common/include/compat/msvc/stdint.h

```diff
@@ -31,8 +31,8 @@
 #ifndef __RARCH_STDINT_H
 #define __RARCH_STDINT_H
 
-#if _MSC_VER && (_MSC_VER < 1600)
```

Comment:
Hmm, still uncertain why this change is needed though.

This seems to indicate MSVC has had stdint.h since 2010.

Reply:
The 1600 value is for Visual Studio 2010 (MSVC++ 10.0), if you want to support 2017 (MSVC++ 14.1) and below, the value needs to be 1911 otherwise compiler errors will occur.

Reply:
@bparker06 What do you make of this change?

Reply:
I'm not sure to understand the question.
This information comes from a list of values depend the Visual Studio versions:

MSVC++ 4.x  _MSC_VER == 1000
MSVC++ 5.0  _MSC_VER == 1100 (Visual Studio 97)
MSVC++ 6.0  _MSC_VER == 1200 (Visual Studio 6.0)
MSVC++ 7.0  _MSC_VER == 1300 (Visual Studio .NET 2002)
MSVC++ 7.1  _MSC_VER == 1310 (Visual Studio .NET 2003)
MSVC++ 8.0  _MSC_VER == 1400 (Visual Studio 2005)
MSVC++ 9.0  _MSC_VER == 1500 (Visual Studio 2008)
MSVC++ 10.0 _MSC_VER == 1600 (Visual Studio 2010)
MSVC++ 11.0 _MSC_VER == 1700 (Visual Studio 2012)
MSVC++ 12.0 _MSC_VER == 1800 (Visual Studio 2013)
MSVC++ 14.0 _MSC_VER == 1900 (Visual Studio 2015)
MSVC++ 14.1 _MSC_VER >= 1910 (Visual Studio 2017)

Reply:
Not sure what is trying to be accomplished here... I was able to compile for VC2015 and 2017 with the current HEAD without applying any patches from this PR... we use our current stdint.h with every compiler version from VC6 all the way up to 2017.

Reply:
That's fine, so you can discard my modification then.
At my side, I had a compilation issue; I will try again to compile without the modification, and if I got the error, I will simply keep my modification locally.
I'm sincerely sorry to have bothered you with this.

Reply:
No need to apologize djipi, in fact we should thank you instead for paving the road towards MSVC2017 support like this.